R/Rsw.R

```diff
@@ -383,6 +390,11 @@ sw_inputDataFromFiles <- function(
   input <- sw_args(dir, files.in, echo = FALSE, quiet = quiet)
 
   res <- .Call(C_onGetInputDataFromFiles, input, quiet)
+
+  if (is.null(res)) {
```

Comment:
Why does commit 362fada5ed00b72178bd61f29215bbecbe0e7020 not initialize `res` to `NULL` in `sw_inputDataFromFiles()` as it does for the other functions?